components/brave_webtorrent/extension/manifest.json

```diff
@@ -7,7 +7,7 @@
     "scripts": ["extension/out/brave_webtorrent_background.bundle.js"],
     "persistent": true
   },
-  "permissions": ["dns", "tabs", "windows", "<all_urls>"],
+  "permissions": ["downloads", "dns", "tabs", "windows", "<all_urls>"],
```

Comment:
cc @diracdeltas for granting a new permission to webtorrent extension.

Reply:
seems ok